packages/sui-mockmock/src/index.js

```diff
@@ -0,0 +1,5 @@
+import HttpMocker from './http'
+
+export {
+  HttpMocker
```

Comment:
This package preserve the API from the motor`s solution ?!

Reply:
It doesn't break anything. Actually there's in the desc of this pr a link to the final implementation and everything's fine :+1: